### ISSUE TITLE
Is there any progress?

### ISSUE THREAD

**Issue by gavinliu6** [togiberlin/ui_ux_handbook#1]
I am looking forward to your updates. So, is it in progress? Or has it been postponed for the time being?

**Comment by Takaya213** [comment]
Would also like to know this.

**Comment by flacial** [comment]
3rd me.

**Comment by jaeminkim-ISC** [comment]
Hi, I would love to get peer-reviewed on this handbook, as well as collaborators. I am planning to release a pre-version for free on Amazon, if you are willing to put a few K words in, I would love to have you as a co-writer. 

https://github.com/hillodesign/user_interface_and_user_experience_design_handbook

**Comment by khushi200701** [comment]
> Hi, I would love to get peer-reviewed on this handbook, as well as collaborators. I am planning to release a pre-version for free on Amazon, if you are willing to put a few K words in, I would love to have you as a co-writer.
> 
> https://github.com/hillodesign/user_interface_and_user_experience_design_handbook

I would like to contribute, how can I help?

**Comment by jaeminkim-ISC** [comment]
> > Hi, I would love to get peer-reviewed on this handbook, as well as collaborators. I am planning to release a pre-version for free on Amazon, if you are willing to put a few K words in, I would love to have you as a co-writer.
> > https://github.com/hillodesign/user_interface_and_user_experience_design_handbook
> 
> I would like to contribute, how can I help?

https://www.amazon.com/User-Interface-Experience-Design-Handbook-ebook/dp/B09PS93M6G/ref=sr_1_1?keywords=user+interface+and+user+experience+design+handbook&qid=1641902369&sprefix=user+interface+and+user%2Caps%2C122&sr=8-1

**Comment by jaeminkim-ISC** [comment]
Need to get started on the second version asap, adding chapters with information on the actual topics of user interface (UI) and user experience (UX).

**Comment by khushi200701** [comment]
Yes, I can get started, can you help me list out a few topics I should start with?
And how long do we want the chapters?

**Comment by togiberlin** [comment]
Hello @hillodesign 
sorry for late reply.

If you want, I can add you as collaborator.
You can then freely edit files according to your liking.

Would this be an option for you?

**Comment by jaeminkim-ISC** [comment]
> Hello @hillodesign sorry for late reply.
> 
> If you want, I can add you as collaborator. You can then freely edit files according to your liking.
> 
> Would this be an option for you?

https://www.amazon.com/dp/B09PS93M6G/ref=cm_sw_em_r_mt_dp_T5N4BQ7R6RWMKZW6WTFJ

**Comment by jaeminkim-ISC** [comment]
> Hello @hillodesign sorry for late reply.
> 
> If you want, I can add you as collaborator. You can then freely edit files according to your liking.
> 
> Would this be an option for you?

Yes, please I would like that very much

**Comment by tikruXX** [comment]
Is there any progress in the meantime?

**Comment by DenysArkan** [comment]
Dead project?

**Comment by jaeminkim-ISC** [comment]
I'll update the handbook to second version. 

https://github.com/Illimag/user_interface_and_user_experience_design_handbook

Sorry to burst your bubble as of January 2024, there are only 100-200 hybrid/on-site job openings on Linkedin in the San Francisco Bay Area Silicon Valley for UI/UX Designers.

1000s of applicants per position.
